neo/Network/RPC/TransactionManager.cs

```diff
@@ -50,11 +49,17 @@ public TransactionManager(RpcClient rpc, UInt160 sender)
         /// <returns></returns>
         public TransactionManager MakeTransaction(byte[] script, TransactionAttribute[] attributes = null, Cosigner[] cosigners = null, long networkFee = 0)
         {
+            byte[] buffer = new byte[sizeof(uint)];
+            using (var random = System.Security.Cryptography.RandomNumberGenerator.Create())
```

Comment:
@shargon, maybe create a generic function `getBytes` for avoiding repeating this `using`.
In addition, the function can size the `sizeof`.

Reply:
I just saw that for `wallet` it is different.